squarelet/core/templatetags/airtable.py

```diff
@@ -1,26 +1,30 @@
-from urllib.parse import urlencode
+# Django
 from django import template
 from django.utils.safestring import mark_safe
 
+# Standard Library
+from urllib.parse import urlencode
+
 register = template.Library()
 
 VERIFICATION_FORM_URL = "https://airtable.com/app93Yt5cwdVWTnqn/pagogIhgB1jZTzq00/form"
```

Comment:
Can we move this to settings? I know this is the only place it's likely to be used, but I worry it's too buried in a template file.